### PR TITLE
musca shutter control - WIP

### PR DESCRIPTION
**Pull request by lspitler**
@wtgee here is our new PR for the shutter work. Let's decide what this PR is meant to do after we test it.

### CODE REVIEW

**Comment by lspitler** [comment]
@wtgee we left this somewhat abruptly today. Let's try and find some time tomorrow to test it out together.

File: conf_files/huntsman.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ dome:
     template_dir: resources/bisque
     driver: bisque
     brand: bisque
+    port: tty.Bluetooth-Incoming-Port
```
Comment (wtgee) [inline conf_files/huntsman.yaml]: `/dev/Tinyos-DevB` is the correct port.

File: huntsman/dome/musca.py [hunk 2]
```diff
@@ -132,12 +131,20 @@ def __str__(self):
 # Private Methods
 ###############################################################################
 
-    def _send_musca_command(self, cmd, log_message=None):
-        """Send command to serial bluetooth device musca."""
+    def _write_musca(self, cmd, log_message=None):
+        """Write command to serial bluetooth device musca."""
+        if log_message is not None:
+            self.logger.info(log_message)
+        self.serial.ser.write(cmd.encode())
```
Comment (wtgee) [inline huntsman/dome/musca.py]: We need to add the `\n` newline to the cmd and then encode.

**Review by wtgee — APPROVED** [review]
Basic quick review based on testing with the actual dome.

**Pull request merged**